packages/cactus-test-plugin-htlc-eth-besu-erc20/src/test/typescript/integration/plugin-htlc-eth-besu-erc20/refund-endpoint.test.ts

```diff
@@ -37,566 +38,281 @@ import TestTokenJSON from "../../../solidity/token-erc20-contract/Test_Token.jso
-    contractName: TestTokenJSON.contractName,
-    keychainId,
-    signingCredential: web3SigningCredential,
-    invocationType: EthContractInvocationType.Call,
-    methodName: "allowance",
-    params: [firstHighNetWorthAccount, hashTimeLockAddress],
-  });
-  t.equal(responseAllowance.callOutput, "10", "callOutput() is 10 OK");
-
-  t.comment("Create new contract for HTLC");
-  const request: NewContractRequest = {
-    contractAddress: hashTimeLockAddress,
-    inputAmount: 10,
-    outputAmount: 1,
-    expiration: timestamp,
-    hashLock,
-    tokenAddress,
-    receiver,
-    outputNetwork: "BTC",
-    outputAddress: "1AcVYm7M3kkJQH28FXAvyBFQzFRL6xPKu8",
-    connectorId,
-    keychainId,
-    web3SigningCredential,
-    gas: estimatedGas,
-  };
-  const res = await api.newContractV1(request);
-  t.equal(res.status, 200, "response status is 200 OK");
-
-  t.comment("Get account balance");
-  const responseBalance2 = await connector.invokeContract({
-    contractName: TestTokenJSON.contractName,
-    keychainId,
-    signingCredential: web3SigningCredential,
-    invocationType: EthContractInvocationType.Call,
-    methodName: "balanceOf",
-    params: [firstHighNetWorthAccount],
-  });
-  t.equal(responseBalance2.callOutput, "90", "balance of account is 90 OK");
-
-  t.comment("Get HTLC Id from DemoHelper");
-  const responseTxId = await connector.invokeContract({
-    contractName: DemoHelperJSON.contractName,
-    keychainId,
-    signingCredential: web3SigningCredential,
-    invocationType: EthContractInvocationType.Call,
-    methodName: "getTxId",
-    params: [
-      firstHighNetWorthAccount,
-      receiver,
-      10,
-      hashLock,
-      timestamp,
-      tokenAddress,
-    ],
-  });
-  t.ok(responseTxId.callOutput, "result is truthy OK");
-  const id = responseTxId.callOutput as string;
-
-  t.comment("Refund HTLC");
-  await timeout(6000);
-  const refundRequest: RefundRequest = {
-    id,
-    web3SigningCredential,
-    connectorId,
-    keychainId,
-  };
-  const resRefund = await api.refundV1(refundRequest);
-  t.equal(resRefund.status, 200, "response status is 200 OK");
-
-  t.comment("Get account balance");
-  const responseFinalBalance = await connector.invokeContract({
-    contractName: TestTokenJSON.contractName,
-    keychainId,
-    signingCredential: web3SigningCredential,
-    invocationType: EthContractInvocationType.Call,
-    methodName: "balanceOf",
-    params: [firstHighNetWorthAccount],
-  });
-  t.equal(
-    responseFinalBalance.callOutput,
-    "100",
-    "balance of account is 100 OK",
-  );
-  t.comment("Get status of HTLC");
-  const resStatus = await api.getSingleStatusV1({
-    id,
-    web3SigningCredential,
-    connectorId,
-    keychainId,
-  });
-  t.equal(resStatus.status, 200, "response status is 200 OK");
-  t.equal(resStatus.data, 2, "the contract status is 2 - Refunded");
-  t.end();
-});
-
-test("Test invalid refund with invalid time", async (t: Test) => {
```

Comment:
@awadhana Is this one of those tests that you broke into two and these other tests are now in some other test file? If yes, please make put the exact path for that file in the commit message as well and some explanation so that people later wondering where these tests have gone can figure it out as easily as possible.